PWGHF/D2H/Tasks/taskLc.cxx

```diff
@@ -37,10 +38,36 @@ struct HfTaskLc {
   Configurable<double> yCandGenMax{"yCandGenMax", 0.5, "max. gen particle rapidity"};
   Configurable<double> yCandRecoMax{"yCandRecoMax", 0.8, "max. cand. rapidity"};
   Configurable<std::vector<double>> binsPt{"binsPt", std::vector<double>{hf_cuts_lc_to_p_k_pi::vecBinsPt}, "pT bin limits"};
+  // ThnSparse for ML outputScores and Vars
```

Comment:
I would add a configurable to switch on / off the THnSparse regardless the ML (see also comment below about the axes)

Reply:
Done